packages/devtools_shared/lib/src/extensions/extension_manager.dart

```diff
@@ -57,13 +58,16 @@ class ExtensionsManager {
     devtoolsExtensions.clear();
 
     if (rootPath != null) {
-      // TODO(kenz): use 'findExtensions' from package:extension_discovery once it
-      // is published.
-      // final extensions = findExtensions(
-      //   'devtools',
-      //   packageConfig: '$rootPath/.dart_tool/package_config.json',
-      // );
-      final extensions = <_Extension>[];
+      final extensions = await findExtensions(
+        'devtools',
+        packageConfig: Uri.parse(
+          path.join(
+            rootPath,
+            '.dart_tool',
+            'package_config.json',
+          ),
+        ),
+      );
       for (final extension in extensions) {
         final config = extension.config;
         if (config is! Map) {
```

Comment:
nit: I'd suggest printing a warning to stderr, if for some reason the extension doesn't work.

Or at-least do it, if the extension is in the root package (`rootPath == extension.rootUri`. If I'm working on a package that contains an extension, I should probably be told if the extension doesn't work. Less important, if the extension is from a dependency.

Reply:
I can't think of a scenario where the root path would be equal to the `extension.rootUir`. For example, if I'm a package author for `package:foo`, my extension would live at `path/to/foo/extension/devtools`, but root uri would be for some app that uses foo (`path/to/some_app_that_depends_on_foo`), not `path/too/foo`